django/chat/templates/chat/modals/presets/presets_form.html

```diff
@@ -82,52 +88,89 @@
             </div>
           </div>
         {% endif %}
-        <div id="error-message">
-          {% if error_message %}<div class="alert alert-danger" role="alert">{{ error_message }}</div>{% endif %}
-        </div>
-      </form>
+        <div id="error-message"
```

Comment:
Could you put this at the top of the form, to be consistent with other error messages in the app?